internal/control_plane/logger/zerolog.go

```diff
@@ -0,0 +1,22 @@
+package logger
+
+import (
+	_ "github.com/prometheus/common/log"
+	"github.com/rs/zerolog"
+	"octavius/internal/config"
+	"os"
+	"time"
+)
+
+var Log *zerolog.Logger
+
+func Setup() {
+	var logInit zerolog.Logger
```

Comment:
logInit is redundant here. We can directly assign the result of `zerolog.New` to `Log`

Reply:
logInit is of type zerolog.Logger, Log is of type pointer of zeroLog.Logger, we are assigning the address of logInit to Log, anyways refactored it a little